lib/app.js

```diff
@@ -42,12 +43,7 @@ class AppView extends React.Component {
   constructor(props) {
     super(props);
 
-    // Metrics
-    Metrics.launch();
```

Comment:
OK I can add that event in until we are confident that we can get what we need from GA 'session' data.